lib/poxa/web_hook/dispatcher.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Poxa.WebHook.Dispatcher do
+  use GenServer
+
+  alias Poxa.WebHook.EventTable
+  alias Poxa.CryptoHelper
+
+  @timeout 1500
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, :ok, [])
```

Comment:
I get this error when I do that `** (FunctionClauseError) no function clause matching in GenServer.start_link/3`